rotkehlchen/tests/unit/test_assets.py

```diff
@@ -144,6 +144,42 @@ def test_asset_identifiers_are_unique_all_lowercased():
         identifier_set.add(asset_id)
 
 
+def test_assets_have_common_keys():
+    """Test that checks that the keys and types of tokens in all_assets.json are correct"""
+    # Issue #2363
```

Comment:
nitpick: no reason to add issue number here, nor the following token. Code is understandable.